SECURITY.md

```diff
@@ -18,3 +20,83 @@ securely.
+
+
+## Trusted Configurations
+
+In NGINX, configuration files, modules, certificate/key pairs, and NGINX JavaScript are all considered trusted sources. This means that any security issues or vulnerabilities that arise due to the loading or execution of these trusted components within NGINX are not considered vulnerabilities. Operators must ensure the security and integrity of these trusted sources.
+
+### Risks of Misconfigurations
+
+While NGINX considers these components trusted, misconfigurations within these files can create significant security vulnerabilities. It is the responsibility of the operator to implement configurations according to best practices, regularly review them, and apply security updates as necessary.
+
+### Data Plane vs Control Plane
+In the context of NGINX, the data plane and control plane serve distinct but complementary roles.
+
+The data plane is responsible for handling the actual traffic that flows through NGINX, including processing client requests, routing them to the appropriate backend servers, and returning responses. This plane directly interacts with user data, making it critical for performance, security, and reliability. NGINX inherently trusts the content and instructions provided by upstream servers. If an upstream server issues a command NGINX is designed to support, NGINX will execute that command, relying on the integrity and security of the upstream server’s content.
+
+The control plane, on the other hand, is concerned with the configuration, management, and orchestration of NGINX itself. The control plane does not directly handle traffic but instead governs how the data plane operates. Misconfigurations or vulnerabilities in the control plane can lead to improper behavior in the data plane, potentially exposing the system to attacks or mismanagement.
+
+## Modules Under Scope
+This applies to all default, standard and recommended modules. For each module, detailed security considerations and potential attack vectors will be identified, alongside recommended configurations to mitigate risks.
+
+## Debug Logging and Core Files
+Debug logs and core files produced by NGINX may contain un-sanitized data, including sensitive information like client requests, server configurations, and private key material. These artifacts are useful for troubleshooting and  must be handled with care to avoid exposing confidential data.
+
+## Fix and Release Process
+
+When an issue is reported it is immediately triaged.  This often takes some time and we will communicate with the reporter as needed.  
+
+Fixes are created and tested by the core team using a Github private fork for security.  If needed, we will invite the reporter to contribute to the fork and help the team solve and validate the solution.
+
+### Released Versions
+
+All security fixes will be applied to all supported stable releases as well as main, as applicable.  See [releases.md]
```

Comment:
Should we mention NGINX Plus here? It is an important version of nginx, and it is not clear which security policy applies to it.

Reply:
I need help with this section for sure, but I'm not sure how to address Plus.  This is specifically about the open source side since it lives in Github. I blur that line elsewhere by referencing the F5 policy.  ¯\_(ツ)_/¯